conf/application.conf

```diff
@@ -19,8 +19,8 @@ include "backend.conf"
 
 appName = help-to-save-reminder
 
-# Default http client
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientV2Module"
```

Comment:
why we are not removing uk.gov.hmrc.play.bootstrap.HttpClientModule

Reply:
Still HttpClientModule is in use.

Reply:
Hi @gajalakshmiseetharaman, yeah, we need to keep it as the service still uses the old HttpClient. Once we completely port to HttpClientV2, we can drop the module here. Otherwise, it wouldn't bind a implementation of HttpClient for service's use. Thanks